src/ui/action-button/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { memo } from 'react';
+import { Link } from 'react-router-dom';
+
+import Spinner from '../spinner';
+
+interface ActionButtonProps {
+  title: string;
+  path: string;
+  isLoading: boolean;
+  disabled: boolean;
+}
```

Comment:
```suggestion
className?: string
}
```

Также добавь пропсу className для позиционирования. Она также необязательно